fundingmanager.go

```diff
@@ -941,6 +1137,38 @@ func (f *fundingManager) waitForFundingConfirmation(completeChan *channeldb.Open
 		return
 	}
 
+	// With the block height and the transaction index known, we can
+	// construct the compact chanID which is used on the network to unique
+	// identify channels.
+	shortChanID := lnwire.ShortChannelID{
```

Comment:
Scratch that. Just confirmed that the current behavior is actually correct. I just pushed a new addition to the interface-level tests (c47047c0b8f50ead8d97f56679674df2b6c8ab3e) to exercise this.